### ISSUE TITLE
About `atomsWithMutation`

### ISSUE THREAD

**Issue by zqran** [jotaijs/jotai-tanstack-query#19]
```js
const [, createBillAtom] = atomsWithMutation(() => ({
  mutationKey: ['someKey'],
  mutationFn: async data => {},
}))

const SomeComponent = () => {
  const [, mutate] = useAtom(createBillAtom)
  
  const handleClick = () => {
    mutate([]) // ---> Must the param here be an array?
  }
  
  return ...
}
```

Related code:

https://github.com/jotai-labs/jotai-tanstack-query/blob/2d0a3321e596e2f8ec53675d2a0867a7e6e4aa6e/src/atomsWithMutation.ts#L43

**Comment by dai-shi** [comment]
Yeah, `action` has to be an array.
We could improve this with Jotai v2 API. It will be breaking change though. Maybe worth it.

**Comment by kalijonn** [comment]
this is fixed in v0.8.0
